dice_ml/explainer_interfaces/dice_KD.py

```diff
@@ -49,7 +49,7 @@ def build_KD_tree(self, data_df_copy, desired_range, desired_class):
         dataset_instance = self.data_interface.prepare_query_instance(
             query_instance=data_df_copy[self.data_interface.feature_names])
 
-        predictions = self.predict_fn(dataset_instance)
+        predictions = self.predict_fn_KD(dataset_instance)
```

Comment:
I'm thinking whether we can have two standard functions inside explainer_base. One can be `predict_model_scores` and the other, `predict_model`. Then all subclasses can use it.